rle-bitpacked.md

```diff
@@ -0,0 +1,78 @@
+# RLE-Bitpacked hybrid encoder
+
+The RLE-Bitpacked hybrid encoder is a parquet-specific encoder that combines two well known encoding strategies,
+[RLE](https://en.wikipedia.org/wiki/Run-length_encoding) and bitpacking. Note that "combine" here means this encoder allows both encodings within the same stream, and, during encoding, it can switch between them.
+
+This encoder is only used to encode integer values that may either represent definition levels, representation levels or ids of dictionary-encoded pages. Note that this encoder supports integers that can be represented in less than 8 bits.
```

Comment:
I thought in V2 this can be used for normal integers?

Reply:
I admit it is the first time I am going through the parquet format, and I am hopping for some help here.

I based this on `Enconding.md`:

```
Note that the RLE encoding method is only supported for the following types of data:

    Repetition and definition levels
    Dictionary indices
    Boolean values in data pages, as an alternative to PLAIN encoding
```
which I can see I forgot the boolean data. Maybe this requires updating the `Encoding.md` also?

Reply:
Maybe we should leave this detail here, and possibly update the encoding.md if needed to specify all places where it could apply

Reply:
Also rereading encodings.md, I might be confused :)

Reply:
Actually, it seems that booleans are just bitpacked and therefore have no header nor RLE encoding.